PointSourceALARA

```diff
@@ -47,4 +47,4 @@ output zone
    number_density
 end
 
-truncation 1e-6
+truncation 5e-15
```

Comment:
Have you tried something in-between?  Does OpenMC have a threshold?  Such a low threshold can become problematic in some scenarios.

Reply:
With 1e-6 ALARA was making too few nuclides and there wasn't much to compare with OpenMC. I can try something in between the two -- 1e-15 was as low as it would go without getting stuck